app/assets/stylesheets/detailsPage.scss

```diff
@@ -51,9 +74,8 @@ h1 {
   text-overflow: ellipsis;
   white-space: nowrap;
   max-width: 250px;
-  margin-left: 5px;
-  margin-right: 5px;
-  margin-top: -5px;
+  margin-left: 1vw;
+  margin-right: 1vw;
   padding-left: 10px;
```

Comment:
Would it be clearer to use the shorthand padding notation here instead? Using "padding 5px 10px" should do the same thing.

Reply:
Yep, good point.